app/views/shared/_cancel.html.erb

```diff
@@ -0,0 +1,9 @@
+<% other_cancel = link || decorated_session&.sp_return_url || profile_path %>
```

Comment:
as long as we're here: maybe move this assignment inside the `else` block? it's not used otherwise